### ISSUE TITLE
Tell client about refused connections.

### ISSUE THREAD

**Issue by laurenceforshaw** [rsanchez-wsu/jfiles#249]
If a connection is refused by the server, inform the client of this, then close the connection.

**Comment by rsanchez-wsu** [comment]
@laurenceforshaw This is very much a good thing for the server to do.  It might something to include in the discussion for #231 as well.  I can see how there might be a refused connection before commands start flowing, but I can also see that there might be a reason for the server to terminate the client connection (think of a situation where a command runs too long and the server terminates the connection to prevent a denial of service).

**Comment by LoganRickert** [comment]
I agree with Sanchez but as far as command wise, I feel that the `error` command should suffice. The server may send an `ERROR timeout`, signaling to the client that something has gone wrong.

That being said, we may want to add an `ABORT` command which mimics the `error` command but is used to report a fatal issue. This command would tell the client to immediately disconnect and report to the user, where `error` would just be an error with the syntax of the command sent. I think this issue raises a valid point that there may need to be distinctions with the types of errors reported.
